LibreNMS/Proc.php

```diff
@@ -43,6 +43,11 @@ class Proc
      */
     private $_synchronous;
 
+    /**
+     * @var int|null hold the exit code, we can only get this on the first process_status after exit
+     */
+    private $_exitcode = null;
```

Comment:
Are these changes supposed to be in here? Seems totally different from the locking side.